thonny/plugins/cpython_frontend/cp_front.py

```diff
@@ -356,6 +356,18 @@ def _create_venv(self, event=None):
             exe_path = os.path.join(path, "bin", "python3")
 
         if os.path.exists(exe_path):
+            proc = subprocess.Popen(
+                [exe_path, "-m", "ensurepip", "--default-pip", "--upgrade"],
```

Comment:
> It appears that Python is installed in the Freedesktop SDK using the --without-ensurepip conf option as seen [here](https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/blob/master/elements/components/python3.bst?ref_type=heads#L20). 

How come `ensurepip` works in this case?

Reply:
In this case I'd prefer using the work-around only for flatpak installations and leave the current procedure as is for other cases.

Reply:
Yeah, that sounds good to me. I wouldn't be surprised if this workaround becomes unnecessary with next release of the Freedesktop Flatpak SDK. I'll add a check that only does this for the Flatpak.